### DISCUSSION TITLE
Request to change the $link: $primary to $link: #485fc7

### DISCUSSION THREAD

**Discussion by zhaolinlau** [buefy/buefy#4082] [Ideas]
### Description

In buefy documentation, the color of `.is-link ` is `#485fc7` which is the same color as in bulma documentation. However, the [_variables.scss](https://github.com/buefy/buefy/blob/dev/src/scss/utils/_variables.scss) in buefy's repo is different with the buefy documentation.

the `$link` in [_variables.scss](https://github.com/buefy/buefy/blob/dev/src/scss/utils/_variables.scss):

```scss
$primary: #7957d5 !default;
$link: $primary !default;
```

This means that the color of a `b-button` with `type="is-primary"` and `type="is-link"` will be the same. So I am here to request to change the $link to `#485fc7`, the default `is-link` color in bulma.

### Why Buefy need this feature

To match with buefy's documentation.

**Comment by wesdevpro** [comment]
I will @zhaolinlau take a look into this to see if changing the link color to the default looks good and makes the docs more readable. Thank you for your suggestion 🙂
